sources/packages/backend/apps/db-migrations/src/sql/DynamicFormConfigurations/Insert-dynamic-form-configurations-2021-2022.sql

```diff
@@ -0,0 +1,77 @@
+-- Insert configurations for student financial aid application form type.
+INSERT INTO
+    sims.dynamic_form_configurations (
+        form_type,
+        program_year_id,
+        offering_intensity,
+        form_definition_name
+    )
+VALUES
+    (
+        'Student financial aid application',
+        (
+            select
+                id
+            from
+                sims.program_years
+            where
+                program_year = '2021-2022'
+        ),
+        'Full Time',
+        'SFAA2021-22'
+    ),
+    (
+        'Student financial aid application',
+        (
+            select
+                id
+            from
+                sims.program_years
+            where
+                program_year = '2021-2022'
+        ),
+        'Part Time',
+        'SFAA2021-22'
+    );
+
+-- Insert configurations for supporting users parent form type.
+INSERT INTO
+    sims.dynamic_form_configurations (
+        form_type,
+        program_year_id,
+        form_definition_name
```

Comment:
Not a blocker, but the inserts could be segregated by the values they are inserting, for instance, supporting users can go together.